### DISCUSSION TITLE
Is `--watchAll` no longer supported in Razzle 4.X?

### DISCUSSION THREAD

**Discussion by ek68794998** [jaredpalmer/razzle#1645] [Q&A] [answered]
I have this in my `package.json`:
```
{
  "name": "...",
  "version": "...",
  "description": "...",
  "scripts": {
    ...
    "test": "razzle test --watchAll=false",
    ...
  },
  ...
}
```

I copied this and all my other scripts from a previous project (call it P1), which ran on Razzle 3.3.8, into a new project (P2) which is running on Razzle 4.0.5. (Both are on Jest 26.) Previously, when I ran `yarn test` in P1, it would run the tests and exit. This was desired as I've had a number of issues with Jest watchers in the past.

However, in P2, when I run `yarn test`, it runs in watch mode, despite the `--watchAll=false` flag. However, `--no-watch` seems to work in Razzle 4.X.

I cannot find this change documented anywhere; is `--no-watch` the new way to be doing this?

**Comment by fivethreeo** [comment, marked as answer]
--no-watch is the way to go now
